pkg/policy/validate.go

```diff
@@ -236,10 +238,13 @@ func Validate(policy kyverno.PolicyInterface, client *dclient.Client, mock bool,
 		var podOnlyMap = make(map[string]bool) //Validate that Kind is only Pod
 		podOnlyMap["Pod"] = true
 		if reflect.DeepEqual(common.GetKindsFromRule(rule), podOnlyMap) && podControllerAutoGenExclusion(policy) {
-			log.Log.V(4).Info("Pod controllers excluded from autogen require adding of preconditions to also exclude the desired controller(s).")
+			msg := "Policies that match Pods apply to all Pods including those managed by controllers excluded " +
```

Comment:
Maybe I had a typo the first time around, but I smoothed this out.:

"Policies that match Pods apply to all Pods including those originating from any controllers excluded from autogen. Use preconditions to exclude the Pods managed by controllers which were excluded from autogen."